doc/how-to/how-to-setup-custom-subdomains.md

```diff
@@ -0,0 +1,87 @@
+# How to setup custom subdomains
+
+## Context 🖼️
+Teams can setup custom subdomains for PlanX so that the service can be accessed via their domain (e.g. https://planningservices.medway.gov.uk/).
+
+This guide will walk through the process of setting a custom domain for a new team. It will require actions by both PlanX developers and council IT teams.
+
+## Process ⚙️
+1. **PlanX** - Provide IT team with documentation listing what's required. This is part of the onboarding process and briefly outlines the process. Documentation - https://www.notion.so/opensystemslab/IT-systems-services-d4f8b88fb9694f33a24411801150c793
+
+2. **PlanX** - Provide IT team with Certificate Signing Request (CSR) if requested. The IT team will need to tell us what to put in [in each of the fields](https://en.wikipedia.org/wiki/Certificate_signing_request#Procedure).
+
+    **How to generate a CSR**
+    ```shell
+    openssl req -new -newkey rsa:2048 -nodes -keyout <TEAM_NAME>.key -out <TEAM_NAME>.csr
```

Comment:
could we also add a quick note under this section of why `rsa:2048` is required based on the error earlier this week?

Reply:
I've clarified the requirements for this in our docs (I had also previously updated the public docs when going back and forth with James).